jest.setup.js

```diff
@@ -13,6 +13,20 @@ if (!chrome.runtime.id) chrome.runtime.id = "history-delete";
 beforeAll(() => {
   // Enable the mocking in tests.
   server.listen();
+  const get = jest.fn();
+  const set = jest.fn();
+  global.chrome = {
```

Comment:
Did you try the polify? actually I think no `chrome` should be in our code. we can use `browser`.

Reply:
That's just for the jest tests. They run in a chrome env. We were already doing something similar in the tests.  
Problem with the webextension-polyfill is that it doesn't not yet support i.e. `...storage.session` (I guess MV3 in general: https://github.com/mozilla/webextension-polyfill/issues/329). [According to this comment](https://github.com/mozilla/webextension-polyfill/issues/329#issuecomment-1188822881) we could just use `chrome.` in all browsers already? I can try that.

Reply:
ok, interesting. so more changes.

Reply:
Using the polyfill for now but ignoring the missing type. eventually we can think about ditching the polyfill if `chrome` just works everywhere. But not now.